Client/Ecosia/UI/EcosiaTheme.swift

```diff
@@ -38,6 +38,7 @@ class EcosiaTheme {
     var newsPlaceholder: UIColor { .Light.Background.secondary }
     
     var actionSheetBackground: UIColor { .Light.Background.primary }
+    var actionSheetCancelButton: UIColor { primaryButtonActive }
```

Comment:
I guess you created this new color that re-uses an existing one (instead of directly using the existing one) just because the Dark color is different from the existing?
Just double checking since it seems a bit strange.

Usually I believe the idea is to keep the naming matching the one used in Figma so there is consistency between designer and engineer, but just figured that from looking at the code. I do like that approach and see that on Figma it says `Light mode/Button/Button Bg Primary Default: #008009;` so would say it is good to use something like `.Light.Button.backgroundPrimary` since it is more general and might be re-used in figma for other things with that name. 

I'm not sure if this is all actually properly defined and would be up for having this discussion separately from this small change to properly define a desired styling 🙂

Reply:
Thanks for the comment @lucaschifino 👍

Correct, I've created the new one to match its dark counterpart.
I was also under the impression of the fact that we should stress with matching out Figma designs in terms of naming. Hence pushed back first time, asking the design team for recommendations.
Feel free to double check the comments history of [this ticket](https://ecosia.atlassian.net/jira/software/c/projects/MOB/boards/40?modal=detail&selectedIssue=MOB-1796).
I ended up utilising the same as for the `primaryButtonActive`, also kept it as is considering the matching in terms of colours in the current Figma designs. Happy to apply the same color without directly inheriting from `primaryButtonActive` thought 💪

Reply:
Makes sense, thanks for sharing the history.

I agree that the behaviour you have shown on [this comment](https://ecosia.atlassian.net/browse/MOB-1796?focusedCommentId=80324) seems to be the expected one. I would guess design just missed replicating it in Figma, so it is good to double check 🙏